spec/models/outgoing_message/template/internal_review_spec.rb

```diff
@@ -46,7 +46,7 @@
 
 
 
-       [ GIVE DETAILS ABOUT YOUR COMPLAINT HERE ] 
+       [ GIVE DETAILS ABOUT YOUR COMPLAINT HERE ]
```

Comment:
This was intentional (https://github.com/mysociety/alaveteli/pull/3045#discussion-diff-52318220), buuut looking at the use of the code at the [commit](7a909cbc4) that introduced the specs I don't see why we can't remove this now.

https://github.com/mysociety/alaveteli/blob/7a909cbc4/app/models/outgoing_message.rb#L122